apps/web/src/common/modules/widgets/_widgets/pie-chart/PieChart.vue

```diff
@@ -58,6 +59,29 @@ const state = reactive({
     chart: null as EChartsType | null,
     chartData: [],
     unit: computed<string|undefined>(() => widgetFrameProps.value.unitMap?.[state.dataField]),
+    chartLegendOption: computed(() => {
+        if (!state.showLegends) return { show: false };
+        const _option: any = {
```

Comment:
Question: Isn’t the type for option provided separately??

Reply:
oh yes. I'll set it. Thanks!